src/app.ts

```diff
@@ -34,12 +34,13 @@ app.use("/", support);
 app.use("/", mailingList);
 
 app.locals.googleTagId = process.env.GOOGLE_TAG_ID;
+app.locals.showTestBanner = process.env.SHOW_TEST_BANNER === "true";
 
-app.use((req: Request, res: Response, next: NextFunction) => {
+app.use((req: Request, res: Response) => {
     res.status(404).render("404.njk");
 });
 
-app.use((err: unknown, req: Request, res: Response, next: NextFunction) => {
+app.use((err: unknown, req: Request, res: Response) => {
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/no-unused-vars -- Error handling middleware must take 4 arguments
app.use((err: unknown, req: Request, res: Response, next: Next Function) => {
```

We cannot remove the extra param for error handling middleware, even if the param is unused.
The linting check can be disabled with a comment.